packages/labs/src/components/omnibox/omnibox.tsx

```diff
@@ -190,6 +199,7 @@ export class Omnibox<T> extends React.Component<IOmniboxProps<T>, IOmniboxState<
 
     private handleQueryChange = (event: React.FormEvent<HTMLInputElement>) => {
         this.setState({ query: event.currentTarget.value });
+        Utils.safeInvoke(this.props.inputProps.onChange, event);
```

Comment:
Ha yeah, I always forget. Would be nice to be able to set `defaultProps` on those generic components!